lib/Mail.php

```diff
@@ -608,6 +618,12 @@ private function renderParams(string $string, string $context): string
             throw new Exception(sprintf('Failed rendering the %s: %s. Please check your twig sandbox security policy or contact the administrator.',
                 $context, substr($e->getMessage(), 0, strpos($e->getMessage(), ' in "__string'))));
         } finally {
+            // Restore the default escaping strategy (HTML) after rendering the subject
+            if ($defaultStrategy !== null) {
```

Comment:
@youwe-emre-can 
Apparently, Phpstan is a bitch... Can't we just move the `$twig = $templatingEngine->getTwigEnvironment(true);` outside (above) the `try` block to resolve this issue completely? If there are any `SecurityError` (the reason we have this `try-catch` block), then it would be on the `createTemplate` and/or `render` statements